keyboards/fancytech/fancyalice66/fancyalice66.c

```diff
@@ -0,0 +1,281 @@
+/* Copyright 2023 Adophoxia <https://github.com/Adophoxia>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#ifdef RGB_MATRIX_ENABLE
+bool rgb_matrix_indicators_kb(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        rgb_matrix_set_color(43, 0xFF, 0xFF, 0xFF);
+    }
+    return true;
+}
+#endif
+
+#if defined(ENCODER_ENABLE)
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) { 
+        return false; 
+    }
+    if (clockwise) {
+        tap_code_delay(KC_VOLU, 10);
+    } else {
+        tap_code_delay(KC_VOLD, 10);
+    }
+    return true;
+}
+#endif
+
+{
```

Comment:
This doesn't belong here.

Reply:
Removed the whole text of `info.json` from after the `#endif`